core/dbt/parser/manifest.py

```diff
@@ -1886,7 +1886,12 @@ def write_manifest(manifest: Manifest, target_path: str, which: Optional[str] =
     write_semantic_manifest(manifest=manifest, target_path=target_path)
 
 
-def parse_manifest(runtime_config, write_perf_info, write, write_json):
+def parse_manifest(
+    runtime_config: RuntimeConfig,
+    write_perf_info: bool,
+    write: bool,
+    write_json: bool,
+) -> Manifest:
```

Comment:
Before 6b63e31a4eee3272409eae0417cbad7d94b5384a, this was where I was calling `merge_from_artifact`. Now this file diff is just adding type annotations and an inline comment.